Modules/Filtering/AnisotropicSmoothing/test/itkGradientAnisotropicDiffusionImageFilterTest2.cxx

```diff
@@ -69,8 +69,9 @@ itkGradientAnisotropicDiffusionImageFilterTest2(int argc, char * argv[])
 {
   if (argc < 3)
   {
-    std::cerr << "Usage: " << itkNameOfTestExecutableMacro(argv) << " InputImage OutputImage\n";
-    return -1;
+    std::cerr << "Missing parameters." << std::endl;
+    std::cerr << "Usage: " << itkNameOfTestExecutableMacro(argv) << " InputImage OutputImage" << std::endl;
```

Comment:
@jhlegarreta I'm sorry to say so but unnecessary use of `std::endl` is frowned upon nowadays. C++ Core Guidelines, April 13, 2023 even says: [Avoid `endl`](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#slio50-avoid-endl)!